lib/rack/multipart/parser.rb

```diff
@@ -22,18 +22,19 @@ def self.create(env)
         tempfile = env['rack.multipart.tempfile_factory'] ||
           lambda { |filename, content_type| Tempfile.new(["RackMultipart", ::File.extname(filename)]) }
         bufsize = env['rack.multipart.buffer_size'] || BUFSIZE
+        params = (env['rack.multipart.params_class'] || QueryParser::DEFAULT.params_class).new
```

Comment:
I don't really want to encourage sticking more stuff in `env`.  Can we just change the signature for `create` to have a default like `initialize` below?